src/App.tsx

```diff
@@ -1,24 +1,220 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { TodosList } from './components/TodosList/TodosList';
 
-const USER_ID = 0;
+const USER_ID = 6937;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState('');
+  const [query, setQuery] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo>();
+  const [isDisabledInput, setIsDisabledInput] = useState(false);
+  const [completedId, setCompletedId] = useState<number[]>([]);
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const waitErrorEnable = () => {
+    setTimeout(() => {
+      setHasError('');
+    }, 3000);
+  };
```

Comment:
Instead - better to pass error message here and manage both adding and removing error here